scripts/dispatch-torque.py

```diff
@@ -42,28 +47,25 @@ def get_output(command):
 
     args = parser.parse_args(sys.argv[2:])
 
-    resource_args = ''
+    resource_args = []
     if args.username != None:
-        resource_args += ' -N codalab-%s' % args.username
+        resource_args.extend(['-N', 'codalab-%s' % args.username])
     if args.request_cpus != None:
-        resource_args += ' -l nodes=1:ppn=%d' % args.request_cpus
+        resource_args.extend(['-l', 'nodes=1:ppn=%d' % args.request_cpus])
     if args.request_memory != None:
-        resource_args += ' -l mem=%d' % int(args.request_memory)
+        resource_args.extend(['-l', 'mem=%d' % int(args.request_memory)])
     if args.request_queue != None:
-        resource_args += ' -q %s' % args.request_queue
+        resource_args.extend(['-q', args.request_queue])
     if args.request_priority != None:
-        resource_args += ' -p %s' % args.request_priority
+        resource_args.extend(['-p', args.request_priority])
 
-    stdout = get_output('qsub -o /dev/null -e /dev/null%s %s' % (resource_args, args.script))
+    stdout = get_output(ssh(['/usr/local/bin/qsub', '-o', '/dev/null', '-e', '/dev/null', '-q', 'nlp'] + resource_args + [args.script]))
```

Comment:
I know specifying absolute path is necessary for `shell=False`, but is it safe to assume that `qsub` lives here? Do you know if `/usr/bin/env qsub` would work here?